common/supplemental/supplementalData.xml

```diff
@@ -1315,6 +1315,7 @@ XXX Code for transations where no currency is involved
 		<language type="arp" scripts="Latn"/>
 		<language type="arq" scripts="Arab"/>
 		<language type="arq" territories="DZ" alt="secondary"/>
+		<language type="ars" scripts="Arab" territories="SA"/>
```

Comment:
This doesn't hurt, but note that these items are overwritten when the file is processed.

Reply:
Done